template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
A URL is created here in the template and sent out to other places (email, slack, PD), and the link will be rendeered there, so the link generated in the templating must be regexp-escaped. If that's not easy with Go templating, I guess we should have a regexp-escaped version of the receiver string ready to insert here.